src/main/_docs/setup/setup-configuration.md

```diff
@@ -98,10 +98,12 @@ Refer to [GitLab using OAuth]({{base}}{{site.links["ide-git-svn"]}}#gitlab-oauth
 Code [samples]({{ base }}{{site.links["devops-project-samples"]}}) allow you to define sample projects that are cloned into a workspace if the user chooses it when creating a new project. You can add your own.
 
 # Workspace Limits
-You can place limits on how users interact with the system to control overall system resource usage. You can define how many workspaces created, RAM consumed, idle timeout, and a variety of other parameters. See "Workspace Limits" in `che.env`.
+You can place limits on how users interact with the system to control overall system resource usage. You can define how many workspaces created, RAM consumed, idle timeout, and a variety of other parameters.
 
 You can also set limits on Docker's allocation of CPU to workspaces, which may be necessary if you have a very dense workspace population where users are competing for limited physical resources.
 
+Workspace idle timeout can be configured in `che.env` , so that inactive workspaces will be shutdown automatically over this length of time in milliseconds. By default, this value is set to 3600000 (1 hour). If set to "0", then workspaces will not be stopped automatically. Currently, keyboard and mouse interactions in IDE, as well as HTTP requests to ws-agent count as activity
```

Comment:
Please change: "Workspace idle timeout can be configured in `che.env` , so that inactive workspaces will be shutdown automatically over this length of time in milliseconds."

to...

The workspace idle timeout can be configured in `che.env`. Workspaces that are inactive for this time will be shutdown automatically.